jaeger-core/src/main/java/io/jaegertracing/reporters/RemoteReporter.java

```diff
@@ -106,7 +106,7 @@ public void close() {
     }
   }
 
-  void flush() {
+  public void flush() {
```

Comment:
As this is now public, add a bit of javadoc: is it blocking? Can it fail? If it fails, what happens to the spans on the buffer?